From 371e461907ffe13eadd35a216d8a56c31291064b Mon Sep 17 00:00:00 2001
From: Taoyu Li <taoyl@microsoft.com>
Date: Fri, 27 Oct 2017 01:37:48 +0000
Subject: [PATCH] Restart vas when reload minigraph

---
 config/main.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config/main.py b/config/main.py
index ed2f131..98af202 100644
--- a/config/main.py
+++ b/config/main.py
@@ -180,6 +180,7 @@ def load_minigraph():
     run_command("service interfaces-config restart", display_cmd=True)
     run_command("service ntp-config restart", display_cmd=True)
     run_command("service rsyslog-config restart", display_cmd=True)
+    run_command("service vas restart", display_cmd=True)
     run_command("service swss restart", display_cmd=True)
     run_command("service bgp restart", display_cmd=True)
     run_command("service teamd restart", display_cmd=True)
-- 
2.9.3

